libs/api/domains/national-registry-x-road/src/dto/nationalRegistryChildGuardianshipInput.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
+export class GetChildGuardianshipInput {
```

Comment:
I would skip the `Get` prefix, but this is just preference.
```suggestion
export class ChildGuardianshipInput {
```